app/components/atomic/galoy-slider-button/galoy-slider-button.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from "react"
+import { View } from "react-native"
+
+import { makeStyles, Slider, Text, useTheme } from "@rneui/themed"
+
+import { GaloyIcon } from "../galoy-icon"
+
+type GaloySliderComponentProps = {
+  disabled?: boolean
+  callback?: () => void
+  text?: string
+}
+
+enum sliderTextValues {
```

Comment:
The slider may be used for other things besides sending (such as swapping). I would make it more generic by having the following props.
`initialText` `disabledText` `slidingText` `completedText` and use these instead of the sliderTextValues enum.

Reply:
Additionally we cannot have text hardcoded in components as they will need to be translated to the relevant languages in the parent component.